root/etc/s6-overlay/s6-rc.d/init-mod-universal-calibre-setup/run

```diff
@@ -35,29 +21,14 @@ CALIBRE_RELEASE="$(cat /CALIBRE_RELEASE)"
 if [ -z ${CALIBRE_RELEASE+x} ]; then
     CALIBRE_RELEASE=$(curl -sX GET "https://api.github.com/repos/kovidgoyal/calibre/releases/latest" \
     | awk '/tag_name/{print $4;exit}' FS='[""]'); \
-fi 
-
+fi
 
-if [ ! -e /usr/bin/calibre-server ] || [ "${CALIBRE_RELEASE}" != "$(cat /config/.CALIBRE_RELEASE || :)" ]; then
-    echo "**** Installing/updating calibre ****"
-    rm -rf /app/calibre
-    mkdir -p \
-        /app/calibre
-    if [ "$(uname -m)" == "x86_64" ]; then
-        curl -o \
-            /tmp/calibre.txz -L \
-            "https://github.com/kovidgoyal/calibre/releases/download/${CALIBRE_RELEASE}/calibre-${CALIBRE_RELEASE:1}-x86_64.txz"
-    elif [ "$(uname -m)" == "aarch64" ]; then
-        curl -o \
-            /tmp/calibre.txz -L \
-            "https://github.com/kovidgoyal/calibre/releases/download/${CALIBRE_RELEASE}/calibre-${CALIBRE_RELEASE:1}-arm64.txz"
-    fi
+if [[ ! -e /usr/bin/calibre-server ]] || [[ "${CALIBRE_RELEASE}" != "$(cat /config/.CALIBRE_RELEASE || :)" ]]; then
     tar xf \
         /tmp/calibre.txz \
         -C /app/calibre
     rm /tmp/calibre.txz
+    echo "Installing Calibre version $(cat /CALIBRE_RELEASE)"
     /app/calibre/calibre_postinstall
     echo "${CALIBRE_RELEASE}" > /config/.CALIBRE_RELEASE
-else
-    echo "**** Calibre already installed, skipping ****"
 fi
```

Comment:
I think this logic would fail if the user restarts the container after a new calibre version is released, but the mod is not built yet.

Might be simpler to just attempt an install if `/tmp/calibre.txz` exists, which would suggest a mod update and after install it is deleted so restarts with no mod update would skip it.

Reply:
Done.